test/util.go

```diff
@@ -3,19 +3,19 @@ package test
 import (
 	"fmt"
 	"image"
-	"image/draw"
 	"image/png"
 	"os"
 	"path/filepath"
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
nit: move this back next to require

Reply:
Fixed. No idea why GoLand occasionally does stupid things like that. 
I swear I fixed it at least twice!